controllers/user/index.js

```diff
@@ -10,13 +11,28 @@ router.use(noCache, noindex, (req, res, next) => {
     next();
 });
 
+router.use((req, res, next) => {
+    let crumbs = [
+        {
+            label: 'Your account',
+            url: req.baseUrl
+        }
+    ];
+    if (req.user) {
+        crumbs = concat(crumbs, {
+            label: req.user.userData.username
```

Comment:
Not sure we should put the email address in the breadcrumb,

Reply:
Good call, moved it to a macro in all the templates instead.
![image](https://user-images.githubusercontent.com/394376/56355552-7b38fb80-61ce-11e9-9f3b-53a34d902743.png)